examples/native-external-references-mockshop/README.md

```diff
@@ -0,0 +1,307 @@
+* Input `MockShop API Endpoint` as **Parameter name** and `apiEndpoint` as **ID**
+* Select **Symbol** as type and mark the parameter as required
+
+5. The next steps will lead you through the process of providing a Contentful access token to the application and specifying the organization to which the application should be assigned.
+
+> **NOTE:** 
+> Make sure the organization ID you select here is of the organization that has access to the Native external references feature.
+
+You now have a basic application that can be enriched with additional information that will enable the example project you downloaded to function properly.
+
+## Building and uploading the app
+
+After creating the app definition, we can take care of uploading the code by running these commands:
+
+```bash
+npm run build
+npm run upload
+```
+
+For more information on environment variables, see [npm run upload-ci](#npm-run-upload-ci)
+
+The interactive CLI will prompt you to provide additional details, such as a CMA endpoint URL. Select **Yes** when prompted if you’d like to activate the bundle after upload.
+
+## Creating resource entities
+
+Executing the provided command in your terminal will generate three extra entities within the app definition, which are described in more detail under the [Entities Overview](#entities-overview) step.
+
+```bash
+npm run create-resource-entities
+```
+
+This will tell Contentful that we want to connect to `MockShop` via the function we uploaded in [Building and uploading the app](#building-and-uploading-the-app) step and that the same function can provide `MockShop:Product` _Resource Types_.
+
+> **NOTE:** 
+> Please configure the .env variables properly before running these scripts
+>  
+> `create-resource-entities` script generates new entities within the system, and prints out a minimal log when the operation has succeeded.
+>  
+> If you would like to list all the previously created entities, you can utilize a similar script that prints out more verbose details: `npm run show-resource-entities` .
+
+## Installing the app
+
+* After you successfully upload your app, install it to an existing space by running the command: `npm run install-app`
+* Select the space and environment in which you would prefer to install the example app from the dialog that appears. You will have to grant access to the space the app will be installed in.
+* After granting access, the configuration screen, which is rendered by the <ConfigScreen /> component, will be displayed. Input your MockShop API URL into the form and proceed to save the changes.
+
+Your example app is now configured and installed.
+
+The form that will save the MockShop API URL when we install the app has been defined in `src/locations/ConfigScreen.tsx`. More information on how configuration screens are set up can be found in [this App Configuration tutorial](https://www.contentful.com/developers/docs/extensibility/app-framework/app-configuration/).
+
+# Code structure
```

Comment:
I'm missing an entire section going over how the `ResourceProvider` and `ResourceEntities` are set up with the helpers available in this project.

Reply:
Alright, added a small section to make sure we are covering everything.